### DISCUSSION TITLE
WSA won't start

### DISCUSSION THREAD

**Discussion by rafabra988** [MustardChef/WSABuilds#174] [General]
anyone knows how can i fix this? i've tried reinstall, update, etc. and nothing seems to work

https://github.com/MustardChef/WSABuilds/assets/44869209/9227bc21-f487-4f6f-93a5-873eb1ebe4f0

**Comment by AwesomeKalin** [comment]
Try using the option that keeps WSA always on. This made it work for me. (You don't actually need to keep it always on, just switch it off after using it)

**Reply by rafabra988** [reply]
i end up uninstalling and installing it again, and it did the trick, but thanks anyway, next this happens again i'll try it
